dts/bindings/gpio/infineon,xmc4xxx-gpio.yaml

```diff
@@ -0,0 +1,21 @@
+description: INFINEON XMC4XXX GPIO PORT node
+
+compatible: "infineon,xmc4xxx-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    "#gpio-cells":
+      const: 2
+
+    port-id:
```

Comment:
Can't you easily "compute" `port-id` by doing simple math on the the REG_ADDR?  as such, I think you should remove `port-id` and just make a macro in the code to do the math.

Reply:
> Can't you easily "compute" `port-id` by doing simple math on the the REG_ADDR? as such, I think you should remove `port-id` and just make a macro in the code to do the math.

yes true, will update.

Reply:
I removed port-id binding in the latest version and used DT_HAS_xx_ENABLED for gpio and interrupt controller.